### DISCUSSION TITLE
Problems With Location 23.4.0 Qt 6.5.3

### DISCUSSION THREAD

**Discussion by 520kaymann** [Stellarium/stellarium#3632] [General]
I want to plan ahead for the upcoming solar eclipse. I enter in my eclipse watching latitude, longitude, elevation, Special Location Name, DST info and Special Time Zone. Upon saving to list my home location magically replaces my just entered lat and lon. I can go to the list and find my special location name, click on it and what do know my home lat and lon are saved there too instead of eclipse location lat and lon... Nothing I do seems to get the lat and lon to stick to my just entered numbers... Bug in software or bug in me?

**Comment by 520kaymann** [comment]
Well to finish this off: I have a custom landscape in which I added the Latitude, Longitude and Elevation. This is what was overriding my newly entered values. I removed the "home" values from the custom landscape ini and all worked as expected. The missing values did not affect Stellarium's ability to properly enter the values from the location table when switching back to my "Home" location.

**Reply by gzotti** [reply]
If you load a landscape, then set other coordinates, they are used.
If you set coordinates, then load a landscape with location AND the option "[x] Load location from landscape" active, landscape location is loaded. Same may be when storing a default location, and also storing a default landscape. On startup, its location will be used IF you have the option "[x] Load location from landscape" active. So, best thing is to use a landscape with its correct coordinates, or choose the right setting of the option "[x] Load location from landscape".

**Comment by 520kaymann** [comment]
> If you load a landscape, then set other coordinates, they are used. If you set coordinates, then load a landscape with location AND the option "[x] Load location from landscape" active, landscape location is loaded. Same may be when storing a default location, and also storing a default landscape. On startup, its location will be used IF you have the option "[x] Load location from landscape" active. So, best thing is to use a landscape with its correct coordinates, or choose the right setting of the option "[x] Load location from landscape".
 I got it worked out but not true... my landscape is loaded at startup and anything I tried to enter afterwards was overwritten by the lon, lat, elev afterwards even as I tabbed out of those data boxes...
